spec/support/downloads_helper.rb

```diff
@@ -17,16 +17,10 @@ def downloaded_content
     File.read(downloaded_filename)
   end
 
-  def with_empty_downloads_folder
-    remove_downloaded_files
-    yield
-    remove_downloaded_files
-  end
-
   private
 
   def downloaded_filenames
-    Dir[DownloadsHelper.path.join("*")]
+    Dir[DownloadsHelper.path.join("*")].reject { |f| File.directory?(f) }
```

Comment:
Thanks for explaining. What do you think about this?
```suggestion
    Dir[DownloadsHelper.path.join("*")].select { |f| File.file?(f) }
```

Reply:
I agree it's much clearer to pick up the file name, instead of rejecting the directory path. Thanks for this :pray: